package.json

```diff
@@ -41,12 +48,14 @@
     "husky": "^0.14.3",
     "lint-staged": "^4.1.1",
     "mocha": "^3.5.0",
-    "prettier-eslint-cli": "~4.6.0",
+    "prettier": "^1.9.2",
     "react": "^15.6.2",
     "react-dom": "^15.6.0",
     "rimraf": "^2.6.2",
     "ts-loader": "^3.0.2",
-    "ts-node": "^3.3.0"
+    "ts-node": "^3.3.0",
+    "tslint": "^5.8.0",
+    "typescript": "^2.6.2"
```

Comment:
So, all this time the TS part of the build was using the local machine's typescript installation!

Reply:
Doesn't look like it, though that is probably entirely by accident!
```
$ resin-components/ (255-dropdown-color) ll node_modules | grep typescript
drwxr-xr-x   15 lucianbuzzo  staff   510B 20 Dec 17:33 typescript/
drwxr-xr-x    8 lucianbuzzo  staff   272B 20 Dec 17:33 typescript-eslint-parser/
```

I did some digging and it looks like `prettier-eslint` installs typescript as a dependency.

Reply:
Oh, that's cool(er) then :)